### ISSUE TITLE
Internal Server Error when searching youtube music

### ISSUE THREAD

**Issue by tmuras** [raveberry/raveberry#99]
Run on Raspberry Pi 4, Ubuntu 20.10, local installation run with "raveberry run".

```
log.log_response:222 ERROR  Internal Server Error: /ajax/musiq/request_music/
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/dist-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.8/dist-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.8/dist-packages/raveberry/core/musiq/musiq.py", line 187, in request_music
    successful, message, queue_key = self.do_request_music(
  File "/usr/local/lib/python3.8/dist-packages/raveberry/core/musiq/musiq.py", line 138, in do_request_music
    provider.request(
  File "/usr/local/lib/python3.8/dist-packages/raveberry/core/musiq/music_provider.py", line 100, in request
    if not self.check_available():
  File "/usr/local/lib/python3.8/dist-packages/raveberry/core/musiq/youtube.py", line 174, in check_available
    self.info_dict = self.info_dict["entries"][0]
IndexError: list index out of range
runserver.log_action:154 ERROR  HTTP POST /ajax/musiq/request_music/ 500 [2.45, 192.168.2.101:52322]
runserver.log_action:141 INFO  HTTP POST /ajax/musiq/set_repeat/ 200 [0.12, 192.168.2.101:52322]
log.log_response:222 ERROR  Internal Server Error: /ajax/musiq/request_music/
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/dist-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.8/dist-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.8/dist-packages/raveberry/core/musiq/musiq.py", line 187, in request_music
    successful, message, queue_key = self.do_request_music(
  File "/usr/local/lib/python3.8/dist-packages/raveberry/core/musiq/musiq.py", line 138, in do_request_music
    provider.request(
  File "/usr/local/lib/python3.8/dist-packages/raveberry/core/musiq/music_provider.py", line 100, in request
    if not self.check_available():
  File "/usr/local/lib/python3.8/dist-packages/raveberry/core/musiq/youtube.py", line 174, in check_available
    self.info_dict = self.info_dict["entries"][0]
IndexError: list index out of range
runserver.log_action:154 ERROR  HTTP POST /ajax/musiq/request_music/ 500 [1.76, 192.168.2.101:52322]

```

**Comment by tmuras** [comment]
```
IndexError at /ajax/musiq/request_music/
list index out of range

Request Method: POST
Request URL: http://192.168.2.235:8080/ajax/musiq/request_music/
Django Version: 2.2.17
Python Executable: /usr/bin/python3
Python Version: 3.8.6
Python Path: ['/usr/local/lib/python3.8/dist-packages/raveberry', '/usr/lib/python38.zip', '/usr/lib/python3.8', '/usr/lib/python3.8/lib-dynload', '/usr/local/lib/python3.8/dist-packages', '/usr/lib/python3/dist-packages']
Server time: Sat, 9 Jan 2021 20:23:47 +0100
Installed Applications:
['core.apps.CoreConfig',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'watson',
 'compressor',
 'sass_processor',
 'channels']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'core.user_manager.SimpleMiddleware',
 'watson.middleware.SearchContextMiddleware']


Traceback:

File "/usr/local/lib/python3.8/dist-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/usr/local/lib/python3.8/dist-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/lib/python3.8/dist-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/usr/local/lib/python3.8/dist-packages/raveberry/core/musiq/musiq.py" in request_music
  187.         successful, message, queue_key = self.do_request_music(

File "/usr/local/lib/python3.8/dist-packages/raveberry/core/musiq/musiq.py" in do_request_music
  138.                 provider.request(

File "/usr/local/lib/python3.8/dist-packages/raveberry/core/musiq/music_provider.py" in request
  100.             if not self.check_available():

File "/usr/local/lib/python3.8/dist-packages/raveberry/core/musiq/youtube.py" in check_available
  174.             self.info_dict = self.info_dict["entries"][0]

Exception Type: IndexError at /ajax/musiq/request_music/
Exception Value: list index out of range
Request information:
USER: admin

GET: No GET data

POST:
query = 'blackmore night'
playlist = 'false'
platform = 'youtube'
csrfmiddlewaretoken = 'kUoFoZZQATkSluW2rAhMgx9w8j0HpRZ7l410tT0Vq7mqegfjBM3n2zGa6LZDuCPF'

FILES: No FILES data

COOKIES:
csrftoken = '6PJHjnsbr3CntXzVSAUKv9vbtBRw3i6P7Zm2ohtghhEVmJSc2MGlhb2Pr3Qs83Wn'
platform = 'youtube'
sessionid = '4b8pjyjb1ioch3ubjiwbl9x1q6b1j3n1'
ignore_updates = ''

META:
CONTENT_LENGTH = '138'
CONTENT_TYPE = 'application/x-www-form-urlencoded; charset=UTF-8'
CSRF_COOKIE = '6PJHjnsbr3CntXzVSAUKv9vbtBRw3i6P7Zm2ohtghhEVmJSc2MGlhb2Pr3Qs83Wn'
HTTP_ACCEPT = '*/*'
HTTP_ACCEPT_ENCODING = 'gzip, deflate'
HTTP_ACCEPT_LANGUAGE = 'en-US,en;q=0.5'
HTTP_CONNECTION = 'keep-alive'
HTTP_COOKIE = 'csrftoken=6PJHjnsbr3CntXzVSAUKv9vbtBRw3i6P7Zm2ohtghhEVmJSc2MGlhb2Pr3Qs83Wn; platform=youtube; sessionid=4b8pjyjb1ioch3ubjiwbl9x1q6b1j3n1; ignore_updates='
HTTP_DNT = '1'
HTTP_HOST = '192.168.2.235:8080'
HTTP_ORIGIN = 'http://192.168.2.235:8080'
HTTP_REFERER = 'http://192.168.2.235:8080/musiq/'
HTTP_USER_AGENT = 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:84.0) Gecko/20100101 Firefox/84.0'
HTTP_X_REQUESTED_WITH = 'XMLHttpRequest'
PATH_INFO = '/ajax/musiq/request_music/'
QUERY_STRING = ''
REMOTE_ADDR = '192.168.2.101'
REMOTE_HOST = '192.168.2.101'
REMOTE_PORT = 52322
REQUEST_METHOD = 'POST'
SCRIPT_NAME = ''
SERVER_NAME = '192.168.2.235'
SERVER_PORT = '8080'
wsgi.multiprocess = True
wsgi.multithread = True

Settings:
Using settings module main.settings
ABSOLUTE_URL_OVERRIDES = {}
ADMINS = []
ALLOWED_HOSTS = ['*']
APPEND_SLASH = True
ASGI_APPLICATION = 'main.routing.APPLICATION'
AUTHENTICATION_BACKENDS = ['django.contrib.auth.backends.ModelBackend']
AUTH_PASSWORD_VALIDATORS = '********************'
AUTH_USER_MODEL = 'auth.User'
BASE_DIR = '/usr/local/lib/python3.8/dist-packages/raveberry'
CACHES = {'default': {'BACKEND': 'django.core.cache.backends.locmem.LocMemCache'}}
CACHE_MIDDLEWARE_ALIAS = 'default'
CACHE_MIDDLEWARE_KEY_PREFIX = '********************'
CACHE_MIDDLEWARE_SECONDS = 600
CHANNEL_LAYERS = {'default': {'BACKEND': 'channels_redis.core.RedisChannelLayer', 'CONFIG': {'hosts': [('127.0.0.1', 6379)], 'capacity': 1500, 'expiry': 10}}}
COMPRESSORS = {'css': 'compressor.css.CssCompressor', 'js': 'compressor.js.JsCompressor'}
COMPRESS_CACHEABLE_PRECOMPILERS = '()'
COMPRESS_CACHE_BACKEND = 'default'
COMPRESS_CACHE_KEY_FUNCTION = '********************'
COMPRESS_CLEAN_CSS_ARGUMENTS = ''
COMPRESS_CLEAN_CSS_BINARY = 'cleancss'
COMPRESS_CLOSURE_COMPILER_ARGUMENTS = ''
COMPRESS_CLOSURE_COMPILER_BINARY = 'java -jar compiler.jar'
COMPRESS_CSS_HASHING_METHOD = 'mtime'
COMPRESS_DATA_URI_MAX_SIZE = 1024
COMPRESS_DEBUG_TOGGLE = None
COMPRESS_ENABLED = False
COMPRESS_FILTERS = {'js': ['compressor.filters.jsmin.JSMinFilter'], 'css': ['compressor.filters.css_default.CssAbsoluteFilter']}
COMPRESS_JINJA2_GET_ENVIRONMENT = <function CompressorConf.JINJA2_GET_ENVIRONMENT at 0xffff908783a0>
COMPRESS_MINT_DELAY = 30
COMPRESS_MTIME_DELAY = 10
COMPRESS_OFFLINE = False
COMPRESS_OFFLINE_CONTEXT = {'STATIC_URL': '/static/'}
COMPRESS_OFFLINE_MANIFEST = 'manifest.json'
COMPRESS_OFFLINE_TIMEOUT = 31536000
COMPRESS_OUTPUT_DIR = 'CACHE'
COMPRESS_PARSER = 'compressor.parser.AutoSelectParser'
COMPRESS_PRECOMPILERS = '()'
COMPRESS_REBUILD_TIMEOUT = 2592000
COMPRESS_ROOT = '/usr/local/lib/python3.8/dist-packages/raveberry/static'
COMPRESS_STORAGE = 'compressor.storage.CompressorFileStorage'
COMPRESS_TEMPLATE_FILTER_CONTEXT = {'STATIC_URL': '/static/'}
COMPRESS_URL = '/static/'
COMPRESS_URL_PLACEHOLDER = '/__compressor_url_placeholder__/'
COMPRESS_VERBOSE = False
COMPRESS_YUGLIFY_BINARY = 'yuglify'
COMPRESS_YUGLIFY_CSS_ARGUMENTS = '--terminal'
COMPRESS_YUGLIFY_JS_ARGUMENTS = '--terminal'
COMPRESS_YUI_BINARY = 'java -jar yuicompressor.jar'
COMPRESS_YUI_CSS_ARGUMENTS = ''
COMPRESS_YUI_JS_ARGUMENTS = ''
CSRF_COOKIE_AGE = 31449600
CSRF_COOKIE_DOMAIN = None
CSRF_COOKIE_HTTPONLY = False
CSRF_COOKIE_NAME = 'csrftoken'
CSRF_COOKIE_PATH = '/'
CSRF_COOKIE_SAMESITE = 'Lax'
CSRF_COOKIE_SECURE = False
CSRF_FAILURE_VIEW = 'core.util.csrf_failure'
CSRF_HEADER_NAME = 'HTTP_X_CSRFTOKEN'
CSRF_TRUSTED_ORIGINS = []
CSRF_USE_SESSIONS = False
DATABASES = {'default': {'ENGINE': 'django.db.backends.sqlite3', 'NAME': '/usr/local/lib/python3.8/dist-packages/raveberry/db.sqlite3', 'OPTIONS': {'timeout': 20}, 'ATOMIC_REQUESTS': False, 'AUTOCOMMIT': True, 'CONN_MAX_AGE': 0, 'TIME_ZONE': None, 'USER': '', 'PASSWORD': '********************', 'HOST': '', 'PORT': '', 'TEST': {'CHARSET': None, 'COLLATION': None, 'NAME': None, 'MIRROR': None}}}
DATABASE_ROUTERS = []
DATA_UPLOAD_MAX_MEMORY_SIZE = 2621440
DATA_UPLOAD_MAX_NUMBER_FIELDS = 1000
DATETIME_FORMAT = 'N j, Y, P'
DATETIME_INPUT_FORMATS = ['%Y-%m-%d %H:%M:%S', '%Y-%m-%d %H:%M:%S.%f', '%Y-%m-%d %H:%M', '%Y-%m-%d', '%m/%d/%Y %H:%M:%S', '%m/%d/%Y %H:%M:%S.%f', '%m/%d/%Y %H:%M', '%m/%d/%Y', '%m/%d/%y %H:%M:%S', '%m/%d/%y %H:%M:%S.%f', '%m/%d/%y %H:%M', '%m/%d/%y']
DATE_FORMAT = 'N j, Y'
DATE_INPUT_FORMATS = ['%Y-%m-%d', '%m/%d/%Y', '%m/%d/%y', '%b %d %Y', '%b %d, %Y', '%d %b %Y', '%d %b, %Y', '%B %d %Y', '%B %d, %Y', '%d %B %Y', '%d %B, %Y']
DEBUG = True
DEBUG_PROPAGATE_EXCEPTIONS = False
DECIMAL_SEPARATOR = '.'
DEFAULT_CACHE_DIR = '~/Music/raveberry/'
DEFAULT_CHARSET = 'utf-8'
DEFAULT_CONTENT_TYPE = 'text/html'
DEFAULT_EXCEPTION_REPORTER_FILTER = 'django.views.debug.SafeExceptionReporterFilter'
DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
DEFAULT_FROM_EMAIL = 'webmaster@localhost'
DEFAULT_INDEX_TABLESPACE = ''
DEFAULT_TABLESPACE = ''
DISALLOWED_USER_AGENTS = []
DOCKER = False
DOCKER_ICECAST = False
EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
EMAIL_HOST = 'localhost'
EMAIL_HOST_PASSWORD = '********************'
EMAIL_HOST_USER = ''
EMAIL_PORT = 25
EMAIL_SSL_CERTFILE = None
EMAIL_SSL_KEYFILE = '********************'
EMAIL_SUBJECT_PREFIX = '[Django] '
EMAIL_TIMEOUT = None
EMAIL_USE_LOCALTIME = False
EMAIL_USE_SSL = False
EMAIL_USE_TLS = False
FILE_CHARSET = 'utf-8'
FILE_UPLOAD_DIRECTORY_PERMISSIONS = None
FILE_UPLOAD_HANDLERS = ['django.core.files.uploadhandler.MemoryFileUploadHandler', 'django.core.files.uploadhandler.TemporaryFileUploadHandler']
FILE_UPLOAD_MAX_MEMORY_SIZE = 2621440
FILE_UPLOAD_PERMISSIONS = None
FILE_UPLOAD_TEMP_DIR = None
FIRST_DAY_OF_WEEK = 0
FIXTURE_DIRS = []
FORCE_POSTGRES = False
FORCE_SCRIPT_NAME = None
FORMAT_MODULE_PATH = None
FORM_RENDERER = 'django.forms.renderers.DjangoTemplates'
ICECAST_HOST = 'localhost'
IGNORABLE_404_URLS = []
INSTALLED_APPS = ['core.apps.CoreConfig', 'django.contrib.admin', 'django.contrib.auth', 'django.contrib.contenttypes', 'django.contrib.sessions', 'django.contrib.messages', 'django.contrib.staticfiles', 'watson', 'compressor', 'sass_processor', 'channels']
INTERNAL_IPS = []
LANGUAGES = [('af', 'Afrikaans'), ('ar', 'Arabic'), ('ast', 'Asturian'), ('az', 'Azerbaijani'), ('bg', 'Bulgarian'), ('be', 'Belarusian'), ('bn', 'Bengali'), ('br', 'Breton'), ('bs', 'Bosnian'), ('ca', 'Catalan'), ('cs', 'Czech'), ('cy', 'Welsh'), ('da', 'Danish'), ('de', 'German'), ('dsb', 'Lower Sorbian'), ('el', 'Greek'), ('en', 'English'), ('en-au', 'Australian English'), ('en-gb', 'British English'), ('eo', 'Esperanto'), ('es', 'Spanish'), ('es-ar', 'Argentinian Spanish'), ('es-co', 'Colombian Spanish'), ('es-mx', 'Mexican Spanish'), ('es-ni', 'Nicaraguan Spanish'), ('es-ve', 'Venezuelan Spanish'), ('et', 'Estonian'), ('eu', 'Basque'), ('fa', 'Persian'), ('fi', 'Finnish'), ('fr', 'French'), ('fy', 'Frisian'), ('ga', 'Irish'), ('gd', 'Scottish Gaelic'), ('gl', 'Galician'), ('he', 'Hebrew'), ('hi', 'Hindi'), ('hr', 'Croatian'), ('hsb', 'Upper Sorbian'), ('hu', 'Hungarian'), ('hy', 'Armenian'), ('ia', 'Interlingua'), ('id', 'Indonesian'), ('io', 'Ido'), ('is', 'Icelandic'), ('it', 'Italian'), ('ja', 'Japanese'), ('ka', 'Georgian'), ('kab', 'Kabyle'), ('kk', 'Kazakh'), ('km', 'Khmer'), ('kn', 'Kannada'), ('ko', 'Korean'), ('lb', 'Luxembourgish'), ('lt', 'Lithuanian'), ('lv', 'Latvian'), ('mk', 'Macedonian'), ('ml', 'Malayalam'), ('mn', 'Mongolian'), ('mr', 'Marathi'), ('my', 'Burmese'), ('nb', 'Norwegian Bokmål'), ('ne', 'Nepali'), ('nl', 'Dutch'), ('nn', 'Norwegian Nynorsk'), ('os', 'Ossetic'), ('pa', 'Punjabi'), ('pl', 'Polish'), ('pt', 'Portuguese'), ('pt-br', 'Brazilian Portuguese'), ('ro', 'Romanian'), ('ru', 'Russian'), ('sk', 'Slovak'), ('sl', 'Slovenian'), ('sq', 'Albanian'), ('sr', 'Serbian'), ('sr-latn', 'Serbian Latin'), ('sv', 'Swedish'), ('sw', 'Swahili'), ('ta', 'Tamil'), ('te', 'Telugu'), ('th', 'Thai'), ('tr', 'Turkish'), ('tt', 'Tatar'), ('udm', 'Udmurt'), ('uk', 'Ukrainian'), ('ur', 'Urdu'), ('vi', 'Vietnamese'), ('zh-hans', 'Simplified Chinese'), ('zh-hant', 'Traditional Chinese')]
LANGUAGES_BIDI = ['he', 'ar', 'fa', 'ur']
LANGUAGE_CODE = 'en-us'
LANGUAGE_COOKIE_AGE = None
LANGUAGE_COOKIE_DOMAIN = None
LANGUAGE_COOKIE_NAME = 'django_language'
LANGUAGE_COOKIE_PATH = '/'
LOCALE_PATHS = []
LOGGING = {'version': 1, 'disable_existing_loggers': False, 'formatters': {'precise': {'format': '%(asctime)s %(module)s.%(funcName)s:%(lineno)s %(levelname)s  %(message)s', 'datefmt': '%Y-%m-%d %H:%M:%S'}, 'brief': {'format': '%(module)s.%(funcName)s:%(lineno)s %(levelname)s  %(message)s'}}, 'handlers': {'infofile': {'level': 'INFO', 'formatter': 'precise', 'class': 'logging.handlers.RotatingFileHandler', 'filename': '/usr/local/lib/python3.8/dist-packages/raveberry/logs/info.log', 'maxBytes': 15728640, 'backupCount': 10}, 'errorfile': {'level': 'ERROR', 'formatter': 'precise', 'class': 'logging.handlers.RotatingFileHandler', 'filename': '/usr/local/lib/python3.8/dist-packages/raveberry/logs/error.log', 'maxBytes': 15728640, 'backupCount': 10}, 'console': {'level': 'INFO', 'formatter': 'brief', 'class': 'logging.StreamHandler'}, 'docker': {'level': 'ERROR', 'formatter': 'precise', 'class': 'logging.StreamHandler'}}, 'root': {'handlers': ['infofile', 'errorfile', 'console'], 'level': 'DEBUG'}}
LOGGING_CONFIG = None
LOGIN_REDIRECT_URL = '/logged_in/'
LOGIN_URL = '/accounts/login/'
LOGOUT_REDIRECT_URL = '/'
MANAGERS = []
MEDIA_ROOT = ''
MEDIA_URL = ''
MESSAGE_STORAGE = 'django.contrib.messages.storage.fallback.FallbackStorage'
MIDDLEWARE = ['django.middleware.security.SecurityMiddleware', 'django.contrib.sessions.middleware.SessionMiddleware', 'django.middleware.common.CommonMiddleware', 'django.middleware.csrf.CsrfViewMiddleware', 'django.contrib.auth.middleware.AuthenticationMiddleware', 'django.contrib.messages.middleware.MessageMiddleware', 'django.middleware.clickjacking.XFrameOptionsMiddleware', 'core.user_manager.SimpleMiddleware', 'watson.middleware.SearchContextMiddleware']
MIGRATION_MODULES = {}
MOCK = False
MONTH_DAY_FORMAT = 'F j'
MOPIDY_HOST = 'localhost'
NUMBER_GROUPING = 0
PASSWORD_HASHERS = '********************'
PASSWORD_RESET_TIMEOUT_DAYS = '********************'
POSTGRES_HOST = '127.0.0.1'
PREPEND_WWW = False
REDIS_HOST = '127.0.0.1'
ROOT_URLCONF = 'main.urls'
SASS_PROCESSOR_ENABLED = False
SASS_PROCESSOR_ROOT = '/usr/local/lib/python3.8/dist-packages/raveberry/static'
SECRET_KEY = '********************'
SECURE_BROWSER_XSS_FILTER = False
SECURE_CONTENT_TYPE_NOSNIFF = False
SECURE_HSTS_INCLUDE_SUBDOMAINS = False
SECURE_HSTS_PRELOAD = False
SECURE_HSTS_SECONDS = 0
SECURE_PROXY_SSL_HEADER = None
SECURE_REDIRECT_EXEMPT = []
SECURE_SSL_HOST = None
SECURE_SSL_REDIRECT = False
SERVER_EMAIL = 'root@localhost'
SESSION_CACHE_ALIAS = 'default'
SESSION_COOKIE_AGE = 1209600
SESSION_COOKIE_DOMAIN = None
SESSION_COOKIE_HTTPONLY = True
SESSION_COOKIE_NAME = 'sessionid'
SESSION_COOKIE_PATH = '/'
SESSION_COOKIE_SAMESITE = 'Lax'
SESSION_COOKIE_SECURE = False
SESSION_ENGINE = 'django.contrib.sessions.backends.db'
SESSION_EXPIRE_AT_BROWSER_CLOSE = False
SESSION_FILE_PATH = None
SESSION_SAVE_EVERY_REQUEST = False
SESSION_SERIALIZER = 'django.contrib.sessions.serializers.JSONSerializer'
SETTINGS_MODULE = 'main.settings'
SHORT_DATETIME_FORMAT = 'm/d/Y P'
SHORT_DATE_FORMAT = 'm/d/Y'
SIGNING_BACKEND = 'django.core.signing.TimestampSigner'
SILENCED_SYSTEM_CHECKS = []
SONGS_CACHE_DIR = '/root/Music/raveberry/'
STATICFILES_DIRS = []
STATICFILES_FINDERS = "('django.contrib.staticfiles.finders.FileSystemFinder', 'django.contrib.staticfiles.finders.AppDirectoriesFinder', 'compressor.finders.CompressorFinder', 'sass_processor.finders.CssFinder')"
STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.StaticFilesStorage'
STATIC_ROOT = '/usr/local/lib/python3.8/dist-packages/raveberry/static'
STATIC_URL = '/static/'
TEMPLATES = [{'BACKEND': 'django.template.backends.django.DjangoTemplates', 'DIRS': ['/usr/local/lib/python3.8/dist-packages/raveberry/templates/'], 'APP_DIRS': True, 'OPTIONS': {'context_processors': ['django.template.context_processors.debug', 'django.template.context_processors.request', 'django.contrib.auth.context_processors.auth', 'django.contrib.messages.context_processors.messages']}}]
TEST_CACHE_DIR = '/usr/local/lib/python3.8/dist-packages/raveberry/test_cache/'
TEST_NON_SERIALIZED_APPS = []
TEST_RUNNER = 'django.test.runner.DiscoverRunner'
THOUSAND_SEPARATOR = ','
TIME_FORMAT = 'P'
TIME_INPUT_FORMATS = ['%H:%M:%S', '%H:%M:%S.%f', '%H:%M']
TIME_ZONE = 'Europe/Berlin'
USE_I18N = True
USE_L10N = True
USE_THOUSAND_SEPARATOR = False
USE_TZ = True
USE_X_FORWARDED_HOST = False
USE_X_FORWARDED_PORT = False
VERSION = '0.7.7'
WSGI_APPLICATION = 'main.wsgi.application'
X_FRAME_OPTIONS = 'SAMEORIGIN'
YEAR_MONTH_FORMAT = 'F Y'


You're seeing this error because you have DEBUG = True in your
Django settings file. Change that to False, and Django will
display a standard page generated by the handler for this status code.

```

**Comment by raveberry** [comment]
Thanks a lot for the detailed error log.
I can confirm the issue, some search queries failed with this message while others worked fine.
This was a problem with `youtube-dl` not returning any results for the query, I assume it is related to https://github.com/ytdl-org/youtube-dl/issues/27604.

This problem was fixed in 2021.01.16, so simply update youtube-dl:
`pip3 install -U youtube-dl`
